libraries/AC_CustomControl/AC_CustomControl.cpp

```diff
@@ -8,14 +8,15 @@
 #include "AC_CustomControl_Backend.h"
 // #include "AC_CustomControl_Empty.h"
 #include "AC_CustomControl_PID.h"
+#include "AC_CustomControl_LQR.h"
 #include <GCS_MAVLink/GCS.h>
 
 // table of user settable parameters
 const AP_Param::GroupInfo AC_CustomControl::var_info[] = {
     // @Param: _TYPE
     // @DisplayName: Custom control type
     // @Description: Custom control type to be used
```

Comment:
```suggestion
    // @Description: Custom control type to be used: See README.md file
```
you need to add a section in the README file for LQR...maybe with a wikipedia link to it also

Reply:
Hello @Hwurzburg, readme changes have been pushed. Kindly review them